src/gui/guiFuncs.cpp

```diff
@@ -316,6 +316,8 @@ void DissolveWindow::updateStatusBar()
     }
     else if (!dissolveIterating_ && elapsedTimer_.secondsElapsed() > 0.0)
     {
+        statusLabel_->setText("Idle");
```

Comment:
Just a small suggestion, since this reflects when a simulation has been stopped after being actively run in the current GUI session.
```suggestion
        statusLabel_->setText("Stopped");
```